src/routes/notificationRoute.js

```diff
@@ -0,0 +1,76 @@
+import { Router } from 'express';
+import Notifications from '../controllers/notificationController';
+import Auth from '../middlewares/auth';
+
+const router = Router();
+
+/**
+ * @swagger
+ * /notification:
+ *   get:
+ *     tags:
+ *       - Notifications
+ *     security:
+ *       - bearerAuth: []
+ *     name: notification
+ *     summary: Get all notifications
+ *     consumes:
+ *       - application/json
+ *     responses:
+ *       200:
+ *             description: notifications succcessfuly found.
+ *       401:
+ *             description: unauthorized.
+ *       500:
+ *             description: server error.
+ * */
+router.get('/', Auth.adminAuth, Notifications.getNotifications);
```

Comment:
Why did you use adminAuth? is an only admin who can see the notifications?

Reply:
no It's because we named badly the variable but this adminAuth is the authentication of all the users I think I am going Create a bug fix story to rename this valiable as it touches different routes.

Reply:
Better name it right. @James-nyagatare